From 970e64d24189c6ad298e42dd0219ced97cc5b373 Mon Sep 17 00:00:00 2001
From: MaJun <majun258@linux.alibaba.com>
Date: Tue, 28 Jan 2020 15:16:16 +0800
Subject: [PATCH] package/strace: Update the strace version

Update the strace version from 5.0  to 5.4,
because 5.4 already support the csky cpu

Signed-off-by: MaJun <majun258@linux.alibaba.com>
---
 package/strace/strace.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/strace/strace.mk b/package/strace/strace.mk
index 81eb03f..aa3ddd6 100644
--- a/package/strace/strace.mk
+++ b/package/strace/strace.mk
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-STRACE_VERSION = 5.0
+STRACE_VERSION = 5.4
 STRACE_SOURCE = strace-$(STRACE_VERSION).tar.xz
 STRACE_SITE = https://strace.io/files/$(STRACE_VERSION)
 STRACE_LICENSE = LGPL-2.1+
-- 
1.8.3.1

